src/service/viewport-impl.js

```diff
@@ -645,9 +645,6 @@ export class ViewportBindingNatural_ {
     if (doc./*OK*/scrollingElement) {
       return doc./*OK*/scrollingElement;
     }
-    if (doc.body) {
```

Comment:
We may need to check the platform here - I looked around. This will definitely cause many problems.